frontend/src/components/postContent/index.js

```diff
@@ -87,11 +107,54 @@ class PostContent extends Component {
     this.setState({ editorState: eState }, () => this.props.callBackSaveData(convertToRaw(eState.getCurrentContent())))
   }
 
+  onToggleInlineStyling = (styling, e) => {
+    e.preventDefault()
+    this.onChange(RichUtils.toggleInlineStyle(this.state.editorState, styling))
+  }
+
+  onToggleBlockStyling = (styling, e) => {
+    e.preventDefault()
+    this.onChange(RichUtils.toggleBlockType(this.state.editorState, styling))
+  }
+
+  getBlockStyle = (block) => {
+    switch (block.getType()) {
+      case 'left':
+        return 'DraftEditor-alignLeft'
+      case 'center':
+        return 'DraftEditor-alignCenter'
+      case 'right':
+        return 'DraftEditor-alignRight'
+      default:
+        return null
+    }
+  }
```

Comment:
Why not do this?

Feels a lot less dry atleast... Right now you're writing DraftEditor-align multiple times

```suggestion
  getBlockStyle = (block) => {
      let blockType = block.getType()
      
      return blockType ? `DraftEditor-align${ blockType[0].toUpperCase() }${ blockType.slice(1) }` : null
  }
```